src/i18n/messages.json

```diff
@@ -184,6 +268,90 @@
         "description": "Globetrotter (covid permettendo), amante delle bibite gassate e climber. Amo le sfide. Mi piace studiare quello che mi interessa. Sono sintetico.",
         "website": "",
         "permalink": "nicpuppa"
+      },
+      "Agny": {
```

Comment:
Please check also this second part, there are the same errors as in the first part